taxonomies/ingredients.txt

```diff
@@ -265,7 +265,7 @@ cs:Kyselina
 da:Syre
 de:Säuerungsmittel
 el:Μέσο οξίνισης
-es:Acidulante, acidulant
+es:Ácido, acidulante, acidulantes
```

Comment:
I have never seen "Acido" instead of "Acidulante". Is this word really used to signal acidulante additives?